app/controllers/application_controller.rb

```diff
@@ -1,5 +1,16 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) do |u|
+      u.permit(:name, :email, :password, :password_confirmation)
+    end
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :email, :password, :password_confirmation, :current_password)
+    end
```

Comment:
- [ ] When I visit your root path, your app displays the image below. But that should not be so. Devise provides a method for ensuring that only logged in users can navigate the app. Kindly add this helper method to this `application_controller`:

  ```
  before_action :authenticate_user!
  ```
  This way when a user is not logged they are redirected to the sign in page

  ![fwel](https://user-images.githubusercontent.com/36542986/195470483-2d78bff0-6616-47b2-a8a6-83f38d3674b9.jpg)